microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -141,9 +145,11 @@ if (sendMessageId) {
 
 ---
 
-4. Make your `stars.jpeg` available from any browser tab
 
-You're probably wondering why, when you pass the `images/stars.jpeg` must you use the `chrome.runtime.getURL` (or `chrome.extension.getURL` while using Manifest V2) API instead of just passing in the relative URL without the extra prefix like in the previous section.  By the way, that extra prefix, returned by `getUrl` with the image attached looks something like the following:
+<!-- ====================================================================== -->
+## Step 4: Make your `stars.jpeg` available from any browser tab
+
+This is why the `images/stars.jpeg` must use the `chrome.runtime.getURL` (or `chrome.extension.getURL` while using Manifest V2) API instead of just passing in the relative URL without the extra prefix like in the previous section.  The extra prefix, returned by `getUrl` with the image attached, should look something like the following:
```

Comment:
```suggestion
When you pass the `images/stars.jpeg`, you must use the `chrome.runtime.getURL` API (or `chrome.extension.getURL` if using Manifest V2) instead of just passing in the relative URL without the extra prefix like in the previous section.  That extra prefix is returned by `getUrl` with the image attached, and looks something like the following:
```